### ISSUE TITLE
Small regressions

### ISSUE THREAD

**Issue by gerald1248** [arnehilmann/markdeck#43]
I'm currently reworking a deck and I'm finding several features that worked in earlier versions that no longer work. One (the speaker notes issue #42) I've already listed, but there are others.

Rather than prepare a zip for each, all of these issues are visible when trying to render https://github.com/gerald1248/kubernetes-open-source-slides with the current build.

The issues I see are:

* parameters for Vega Lite charts are not picked up (the bar chart's bars are wider than specified and the background isn't transparent as specified)
* custom a2sketch shapes are not picked up
* a2sketch (Operator slide) arrow tips are missing
* as mentioned in #42, the speaker notes always appear, can't be toggled off, yet the actual speaker view (with clock etc.) isn't shown

Happy to look more closely + prepare isolated examples soon, but have to revert to `start` script for now.

**Comment by arnehilmann** [comment]
Regarding the background of Vega-lite charts: 
```
    .... 
    "height": 300,
    "mark": "bar",
    "background": "transparent",
    ....
```
does the trick. Did that change from v0.50 to v0.51? 🤔

**Comment by arnehilmann** [comment]
Regarding the width of the bars in the Vega-lite chart: Does the following config helps?
```
    "config": {
        "bar": {
            "binSpacing": 20
        ....
```
(both specifying the overall width [600 here] and a bandSize might be a problem; or the type "temporal" of the x-axis is neither discrete nor continuous? 🤨)

**Comment by gerald1248** [comment]
The issue is that these work perfectly using `start` but no longer using `markdeck` (I'd need to look up the exact versions). Same for the custom shapes (which aren't used right now) and the speaker notes (which as far as I can tell are currently broken). Sorry I didn't have time to prep individual bug reports.

**Comment by gerald1248** [comment]
I've changed the repo linked earlier to one that has both `start` (works as intended) and `markdeck` (doesn't) so it's easier to compare their outputs.

**Comment by arnehilmann** [comment]
Okay, so the switch to the new major versions of both revealjs and pandoc was not as smooth as expected 😬
I fixed the issue with the speaker notes, and found the bug for the custom shapes (my bad).

**Comment by gerald1248** [comment]
That's brilliant, thank you!

**Comment by arnehilmann** [comment]
Please have a look at https://github.com/gerald1248/kubernetes-open-source-slides/pull/1

**Comment by arnehilmann** [comment]
Regarding the disappearing arrow in a2s svgs:
The problem seem to be that the arrow head get defined multiple times, with the same id:
```
<marker id="Pointer" ...
```
using a unique id here, the arrows do _not_ disappear, even when scrolling around; I will have a look into that.

**Comment by arnehilmann** [comment]
fixed with v0.52.4
